### PR TITLE
Find hang in performance tests

### PR DESCRIPTION
**Pull request by jeschu1**

### CODE REVIEW

File: ProjFS.Mac/PrjFSKext/ProviderMessaging.cpp [hunk 1]
```diff
@@ -23,13 +23,15 @@ struct OutstandingMessage
...
 static atomic_uint_least64_t s_nextMessageId;
+static atomic_uint_least64_t s_messagesSent;
 static volatile bool s_isShuttingDown;
 
 
 bool ProviderMessaging_Init()
 {
     LIST_INIT(&s_outstandingMessages);
-    s_nextMessageId = 1;
+    s_nextMessageId = 0;
```
Comment (wilbaker) [inline ProjFS.Mac/PrjFSKext/ProviderMessaging.cpp]: Why was `s_nextMessageId` changed from 1 to 0?

File: ProjFS.Mac/PrjFSKext/ProviderMessaging.cpp [hunk 2]
```diff
@@ -180,10 +188,22 @@ bool ProviderMessaging_TrySendRequestAndWaitForResponse(
...
                    !s_isShuttingDown)
             {
+                if (loopCount > 12)
+                {
+                    KextLog_Error("NO RESPONSE AFTER %d seconds Count: Count:%lld Processed:%lld messageType:%d", loopCount * 5, nextMessageId, s_messagesSent, messageType);
+                }
+                
+                if (s_nextMessageId - s_messagesSent > 1000)
+                {
+                    KextLog_Error("While Loop Message Count: Count:%lld Processed:%lld messageType:%d", s_nextMessageId, s_messagesSent, messageType);
```
Comment (wilbaker) [inline ProjFS.Mac/PrjFSKext/ProviderMessaging.cpp]: In your testing so far have you seen this message get logged?

File: ProjFS.Mac/PrjFSKext/ProviderMessaging.cpp (hunk 1 above)
Comment (jeschu1) [reply ProjFS.Mac/PrjFSKext/ProviderMessaging.cpp]: It could have stayed at 1.

File: ProjFS.Mac/PrjFSKext/ProviderMessaging.cpp (hunk 2 above)
Comment (jeschu1) [reply ProjFS.Mac/PrjFSKext/ProviderMessaging.cpp]: Yes, I believe I saw that Friday.

**Pull request closed without merging**